content/blog/2023-07-19-semgrep.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Linting PromQL expressions using semgrep
+created_at: 2023-07-23
+kind: article
+author_name: Michael Hoffmann (https://github.com/MichaHoffmann)
+---
+
```

Comment:
Feel free to add some note about you https://github.com/prometheus/docs/blob/main/content/blog/2021-11-16-agent.md?plain=1#L8

Reply:
(in 3rd person)